package/yast2.spec

```diff
@@ -139,44 +117,44 @@ Provides:       y2t_menu
 Provides:       yast2-trans-menu
 Obsoletes:      y2t_menu
 Obsoletes:      yast2-trans-menu
-
 # moved here from another packages
-Provides:       yast2-dns-server:/usr/share/YaST2/modules/DnsServerAPI.pm
-Provides:       yast2-installation:/usr/share/YaST2/modules/Hotplug.ycp
-Provides:       yast2-installation:/usr/share/YaST2/modules/HwStatus.ycp
-Provides:       yast2-installation:/usr/share/YaST2/modules/Installation.ycp
-Provides:       yast2-installation:/usr/share/YaST2/modules/Product.ycp
+Provides:       yast2-dns-server:%{_datadir}/YaST2/modules/DnsServerAPI.pm
+Provides:       yast2-installation:%{_datadir}/YaST2/modules/Hotplug.ycp
+Provides:       yast2-installation:%{_datadir}/YaST2/modules/HwStatus.ycp
+Provides:       yast2-installation:%{_datadir}/YaST2/modules/Installation.ycp
+Provides:       yast2-installation:%{_datadir}/YaST2/modules/Product.ycp
 Provides:       yast2-mail-aliases
-Provides:       yast2-network:/usr/share/YaST2/modules/Internet.ycp
-Provides:       yast2-packager:/usr/lib/YaST2/servers_non_y2/ag_anyxml
-
-Requires:       yast2-ruby-bindings >= 3.1.33
-
-Summary:        YaST2 - Main Package
-License:        GPL-2.0
-Group:          System/YaST
+Provides:       yast2-network:%{_datadir}/YaST2/modules/Internet.ycp
+Provides:       yast2-packager:%{_prefix}/lib/YaST2/servers_non_y2/ag_anyxml
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+# for Punycode.rb (bnc#651893) - the idnconv tool is located in
```

Comment:
Um, maybe I'm wrong:

```
# rpm -q --whatprovides /usr/bin/idnconv
bind-utils-9.9.6P1-33.1.x86_64
# zypper se --provides /usr/bin/idnconv
Loading repository data...
Reading installed packages...

S | Name       | Summary                         | Type   
--+------------+---------------------------------+--------
i | bind-utils | Utilities to query and test DNS | package

```

So maybe it would work...